pkg/24.0.8/config.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DOCKER_GIT_COMMIT="e5543d473431b782227f8908005543bb4389b8de"
+
+TEST_OS_IMAGE_NAME=(redhat/ubi9 oraclelinux centos ubuntu rockylinux quay.io/centos/centos)
+TEST_OS_IMAGE_TAG[0]="9.0.0"
+TEST_OS_IMAGE_TAG[1]="7 8"
+TEST_OS_IMAGE_TAG[2]="centos7"
+TEST_OS_IMAGE_TAG[3]="20.04 22.04"
+TEST_OS_IMAGE_TAG[4]="8"
+TEST_OS_IMAGE_TAG[5]="stream8"
```

Comment:
Could we update this list based on latest OS versions we need to support? 

Apologies if this is not the list we refer to for testing purposes, but based on  based on https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-2/?  
rockylinux and oraclelinux would be at 9 now, redhat would be at 9.1 or 9.2 too. 

Same for 24.0.9.

Reply:
updated the versions.